settings/Core.setting.php

```diff
@@ -1037,4 +1037,18 @@
       'none' => ts('None - disable menu'),
     ),
   ),
+  'menubar_color' => array(
+    'group_name' => 'CiviCRM Preferences',
+    'group' => 'core',
+    'name' => 'menubar_position',
+    'type' => 'String',
+    'html_type' => 'color',
+    'default' => '#f2f2f2',
```

Comment:
Overall, making this configurable is awesome, @colemanw.  My only concern is that by defaulting to `#f2f2f2` this only solves the issue for savvy site admins who are probably the ones least confused by the change.  An easy fix might be to default it to `#1b1b1b`, the color of the old menu.  A more ambitious alternative might be to leave this as `#f2f2f2` and then have an upgrade step set `menubar_color` as `#1b1b1b` so new users get bright, fresh Civi and upgrading users have continuity.

Reply:
I've updated the default to `#1b1b1b` - I've been swayed by the feedback to think that consistency is better than a "fresh new look" and having the color be configurable means anyone not happy with it can change it.